common/shlibs

```diff
@@ -4094,3 +4094,120 @@ libkaddressbookmergelibprivate.so.5 kdepim-addons-21.08.3_1
 libkmailconfirmbeforedeleting.so.5 kdepim-addons-21.08.3_1
 libkaddressbookprivate.so.5 kaddressbook-21.08.3_1
 libcodec2.so.1.0 codec2-1.0.1_1
+libvalarrayDict.so root-6.24.06_1
```

Comment:
Those libraries shouldn't be listed here, they are not versioned so there is nothing to gain from that.

Reply:
I see, removed them from the commit